### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by mj-e-boc**
Checked page accessibility and found the following results

- Page titles - No issues found 

- Image text alternatives 

added on line 97(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/index.html#L97) added text alternative for the line 248(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/index.html#L248) added text alternative for the line icon line 232(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/index.html#L232) added text alternative for icon 
 line 264(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/index.html#L264) added image text alternative for icon 

-Text headings - No issues found 

-Color contrast 

- changed background color of button from failing #e305304 to #963903 , line 1350(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/styles.css#L1350), changed background color of border to #993800 line 177(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/styles.css#L177), line 1345(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/styles.css#L1345) , 1346(https://github.com/mj-e-boc/M-J-Portfolio/blob/03e74674bd4aff86f60ec341ca129c1b1de35683/styles.css#L1346) css

-Resize   

- increased project 1 font from 16px to 18px when screen enlarges for better visibility, line 
441(https://github.com/mj-e-boc/M-J-Portfolio/blob/60c63cf19298cba468e414441a2c703bdce723c9/styles.css#L441), line 757(https://github.com/mj-e-boc/M-J-Portfolio/blob/60c63cf19298cba468e414441a2c703bdce723c9/styles.css#L441), line 1052(https://github.com/mj-e-boc/M-J-Portfolio/blob/60c63cf19298cba468e414441a2c703bdce723c9/styles.css#L441)  for better view 
  Increased  main page introduction paragraph text line 437(https://github.com/mj-e-boc/M-J-Portfolio/blob/60c63cf19298cba468e414441a2c703bdce723c9/styles.css#L437), 

-Interaction - No issues found

-Moving content - No issues found

-Multimedia - No issues found 

-The basic structure of the page - schematic throughout,

### CODE REVIEW

**Review by stanleySimeon — APPROVED** [review]
# Hi @mj-e-boc, @mryeminthu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:


# Congratulations! 🎉 

## Highlights
- Correct use of Gitflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -434,6 +434,14 @@ textarea {
     padding: 0;
   }
 
+  main p {
+    font-size: 18px;
+  }
+
+  .card p {
+    font-size: 18px;
```
Comment (Kaiserabbas) [inline styles.css:442]: Kindly consider using unit values in percentages. It will increase responsiveness on your page.

**Review by Kaiserabbas — COMMENTED** [review]
You have done a good job. Kindly consider my comments. Thanks

File: index.html [hunk 2]
```diff
@@ -31,13 +31,13 @@
...
       <ul>
         <li>
-          <a href="">Portfolio</a>
+          <a href="#portfolio">Portfolio</a>
         </li>
         <li>
-          <a href="">About</a>
+          <a href="#aboutme">About</a>
         </li>
         <li>
```
Comment (AbuTalha3) [inline index.html:39]: You have done a great job, everything is as per the given Figma design. Nothing to be changed. It's time to merge.

**Review by AbuTalha3 — COMMENTED** [review]
Good job , Everything is in place. Merge it......

File: styles.css [hunk 3]
```diff
@@ -174,7 +174,7 @@ main ul {
 form button {
   width: 40%;
   margin: auto;
-  background-color: #f55800;
+  background-color: #993800;
```
Comment (Kaiserabbas) [inline styles.css:177]: Kindly check for accessibility issues (Color-pair Contrast Testing).

**Review by AbuTalha3 — COMMENTED** [review]
The project is done very well but it is suggested that you can check for color contrast .

File: styles.css [hunk 4]
```diff
@@ -434,6 +434,14 @@ textarea {
     padding: 0;
   }
 
+  main p {
+    font-size: 18px;
```
Comment (Kaiserabbas) [inline styles.css:438]: Kindly consider using rem for font units.

**Comment by AbuTalha3** [comment]
Check linters please

**Review by Kaiserabbas — COMMENTED** [review]
Kindly check for color-contrast to increase accessibility to your portal

**Pull request merged**